test/test_tree_2d_mhd.jl

```diff
@@ -145,6 +145,21 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_2d_dgsem")
         @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
       end
   end
+
+  @trixi_testset "elixir_mhd_shockcapturing_subcell.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_shockcapturing_subcell.jl"),
+      l2   = [2.9974425783503109e-02, 7.2849646345685956e-02, 7.2488477174662239e-02, 0.0000000000000000e+00, 1.2507971380965512e+00, 1.8929505145499678e-02, 1.2218606317164420e-02, 0.0000000000000000e+00, 3.0154796910479838e-03],
+      linf = [3.2147382412340830e-01, 1.3709471664007811e+00, 1.3465154685288383e+00, 0.0000000000000000e+00, 1.6051257523415284e+01, 3.0564266749926644e-01, 2.3908016329805595e-01, 0.0000000000000000e+00, 1.3711262178549158e-01],
+      tspan = (0.0, 0.003))
+      # Ensure that we do not have excessive memory allocations 
+      # (e.g., from type instabilities) 
+      let 
+        t = sol.t[end] 
+        u_ode = sol.u[end] 
+        du_ode = similar(u_ode) 
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
+      end
```

Comment:
```suggestion
      let 
        t = sol.t[end] 
        u_ode = sol.u[end] 
        du_ode = similar(u_ode) 
        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 15000 
      end
```
Actually, I was very surprised that the test didn't fail after the commit from yesterday:laughing:
The maximum allowed allocation bound for simulations with a custom integrator (e.g. with `SImpleSSPRK`) has to be higher. See [here](https://github.com/trixi-framework/Trixi.jl/pull/1686#issuecomment-1776860956)

Reply:
Ahh, interesting. Cool, I hope it now passes. :D